.env.template

```
@@ -0,0 +1,71 @@
+NODE_ENV=development
+
+# Basic Configuration
+NEXT_PUBLIC_PRIVY_APP_ID=
+NEXT_PUBLIC_CAMPAIGN_INFO_FACTORY=
+NEXT_PUBLIC_PLEDGE_TOKEN=
+NEXT_PUBLIC_PLEDGE_TOKEN_DECIMALS=18
+NEXT_PUBLIC_ITEM_REGISTRY=
+NEXT_PUBLIC_RPC_URL=https://alfajores-forno.celo-testnet.org
+DATABASE_URL=postgres://akashic:akashic@db:5432/akashic_dev
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NEXT_PUBLIC_CAMPAIGN_INFO_FACTORY key

(UnorderedKey)